presto-main/src/main/java/io/prestosql/util/JsonUtil.java

```diff
@@ -91,9 +91,9 @@
 import static io.prestosql.spi.type.RealType.REAL;
 import static io.prestosql.spi.type.SmallintType.SMALLINT;
 import static io.prestosql.spi.type.TinyintType.TINYINT;
-import static io.prestosql.type.JsonType.JSON;
 import static io.prestosql.type.DateTimes.formatTimestamp;
 import static io.prestosql.type.DateTimes.scaleEpochMillisToMicros;
+import static io.prestosql.type.JsonType.JSON;
```

Comment:
Why this moved in your PR?

Reply:
Git getting confused because of a split change across the two commits. I'll update.